pkg/vsphere/compute/placement/random.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package placement
+
+import (
+	"math/rand"
+
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+// RandomHostPolicy chooses a random host on which to power-on a VM.
+type RandomHostPolicy struct {
+	cluster *object.ComputeResource
+}
+
+// NewRandomHostPolicy returns a RandomHostPolicy instance.
+func NewRandomHostPolicy(op trace.Operation, cls *object.ComputeResource) (*RandomHostPolicy, error) {
+	return &RandomHostPolicy{cluster: cls}, nil
+}
+
+// CheckHost always returns false in a RandomHostPolicy.
+func (p *RandomHostPolicy) CheckHost(op trace.Operation, vm *object.VirtualMachine) bool {
+	return false
+}
+
+// RecommendHost recommends a random host on which to place a newly created VM.
```

Comment:
Comment doesn't quite make sense now that a slice of hosts is returned instead of a single host.